sdk/python/feast/feature_view.py

```diff
@@ -61,10 +60,10 @@ class FeatureView(BaseFeatureView):
         ttl: The amount of time this group of features lives. A ttl of 0 indicates that
             this group of features lives forever. Note that large ttl's or a ttl of 0
             can result in extremely computationally intensive queries.
-        batch_source (optional): The batch source of data where this group of features
+        batch_source: The batch source of data where this group of features
```

Comment:
huh did you do this in reverse? the docs suggest we want source instead of batch_source vs stream_source

Reply:
oh these are attributes. Maybe remove the two lines on "deprecated in favor of source"

Reply:
@adchia the `__init__` takes in `source` as a parameter; it then sets `batch_source` and `stream_source` as attributes

Reply:
good catch

Reply:
yep. just remove the two lines here saying "This is deprecated in favor of `source`" ?